hadoop-tools/hadoop-federation-balance/src/test/java/org/apache/hadoop/tools/fedbalance/TestDistCpProcedure.java

```diff
@@ -74,6 +77,9 @@
           new FileEntry(SRCDAT + "/b/c", false)};
   private static String nnUri;
 
+  @Rule
```

Comment:
Maybe add a comment justifying the long timeout.

Reply:
Thanks @goiri for your review. I added a comment for this.